pyogrio/tests/test_geopandas_io.py

```diff
@@ -1765,6 +1765,10 @@ def test_read_invalid_poly_ring(tmp_path, use_arrow, on_invalid, message):
 
 def test_read_multisurface(multisurface_file, use_arrow):
     if use_arrow:
+        # TODO: revisit once https://github.com/geopandas/pyogrio/issues/478
+        # is resolved.
+        pytest.skip("Shapely + GEOS 3.13 crashes in from_wkb for this case")
```

Comment:
Can you move this to an annotation instead of inline in the test function?

Reply:
I'm not sure how to use a fixture in an annotation, and I'm not sure if it is worth the effort to try to get this working for a temporary hack that will only be in use for a few days or weeks at max?

If you know how, feel free...

Reply:
You're right, I wasn't looking at `use_arrow` when I said that.  All good as is.